sql/core/src/test/scala/org/apache/spark/sql/execution/command/AlterTableAddPartitionSuiteBase.scala

```diff
@@ -22,6 +22,20 @@ import org.apache.spark.sql.catalyst.analysis.PartitionsAlreadyExistException
 import org.apache.spark.sql.catalyst.catalog.CatalogTypes.TablePartitionSpec
 import org.apache.spark.sql.internal.SQLConf
 
+/**
+ * This base suite contains unified tests for the `ALTER TABLE .. ADD PARTITION` command that
+ * check V1 and V2 table catalogs. The tests that cannot run for all supported catalogs are
+ * located in more specific test suites:
+ *
+ *   - V2 table catalog tests:
+ *     [[org.apache.spark.sql.execution.command.v2.AlterTableAddPartitionSuite]]
+ *   - V1 table catalog tests:
+ *     [[org.apache.spark.sql.execution.command.v1.AlterTableAddPartitionSuiteBase]]
+ *     - V1 In-Memory catalog:
+ *       [[org.apache.spark.sql.execution.command.v1.AlterTableAddPartitionSuite]]
+ *     - V1 Hive External catalog:
+ *       [[org.apache.spark.sql.hive.execution.command.AlterTableAddPartitionSuite]]
```

Comment:
@dongjoon-hyun It seems that the ref to outer package causes some troubles in generating java docs. I will replace `[[]]` by `''`